File: Python/1013.py
lines = input().split(" ")

c = int(lines.pop())
b = int(lines.pop())
a = int(lines.pop())

maiorAb = (a + b + abs(a - b)) / 2
maior = (c + maiorAb + abs(c - maiorAb)) / 2

print(f"{maior:.0f} eh o maior")

File: JavaScript/1013.js
function maiorAB( a, b ) {
    if( a > b )
        return (a + b + (a - b)) / 2;

    else
        return (a + b + (b - a)) / 2;
}

let input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8');
let lines = input.split('\n');
let valores = lines[0].split(' ');

let A = parseInt(valores.shift());
let B = parseInt(valores.shift());
let C = parseInt(valores.shift());

let parcial = maiorAB(A, B);
let maior = maiorAB(parcial, C);

console.log(maior + " eh o maior");
